themes/brainy/README.md

```diff
@@ -2,7 +2,37 @@
 
 Simple colorful terminal prompt theme (inspired by a number of themes).
 
-![alt text](https://www.lfsystems.xyz/img/brainy-Arch.png)
+Supported on all operating systems.
+
+In constant maintenance and improvement
+
+![alt text](https://www.lfsystems.xyz/img/BrainyTheme.gif)
+
+## Install Theme
+
+### Manually
+
+You can install the theme manually by following these steps:
+Edit your modified config `~/.bashrc` file in order to customize Bash-it, set `BASH_IT_THEME` to the theme name `brainy`.
+
+Examples:
+
+```bash
+# Use the "brainy" theme
+export BASH_IT_THEME="brainy"
+```
+
+### Automatically via terminal
+
+1. You can install the theme automatically using the `sed` command from your Linux or OSX Terminal.
+2. For installation on windows you should use [`Git-Bash`](https://git-for-windows.github.io/) or make sure the terminal emulator you use (ej: cygwin, mintty, etc) has the `sed` command installed.
+
+Command to execute:
+
+```bash
+# Set the "brainy" theme replacing the theme you are using of bash-it
+sed -i 's/'"$BASH_IT_THEME"'/brainy/g' ~/.bashrc
```

Comment:
Hi @nwinkler 
I have updated the readme according to your observations.

I am attentive to your comments.